beacon_node/network/src/sync/network_context.rs

```diff
@@ -502,54 +443,43 @@ impl<T: BeaconChainTypes> SyncNetworkContext<T> {
         Ok(peer_id_to_request_map)
     }
 
-    pub fn range_request_failed(&mut self, request_id: Id) -> Option<RangeRequestId> {
-        let sender_id = self
-            .range_block_components_requests
-            .remove(&request_id)
-            .map(|(sender_id, _info)| sender_id);
-        if let Some(sender_id) = sender_id {
-            debug!(
-                self.log,
-                "Sync range request failed";
-                "request_id" => request_id,
-                "sender_id" => ?sender_id
-            );
-            Some(sender_id)
-        } else {
-            debug!(self.log, "Sync range request failed"; "request_id" => request_id);
-            None
-        }
-    }
-
     /// Received a blocks by range or blobs by range response for a request that couples blocks '
     /// and blobs.
-    pub fn range_block_and_blob_response(
+    pub fn range_block_component_response(
         &mut self,
-        request_id: Id,
-        block_or_blob: BlockOrBlob<T::EthSpec>,
-    ) -> Option<BlocksAndBlobsByRangeResponse<T::EthSpec>> {
-        let Entry::Occupied(mut entry) = self.range_block_components_requests.entry(request_id)
-        else {
+        id: ComponentsByRangeRequestId,
+        block_or_blob: RangeBlockComponent<T::EthSpec>,
```

Comment:
Should we rename this to `block_component` or `range_block_component`?

Reply:
Yeah! 000bc5512